EIPS/eip-6963.md

```diff
@@ -46,9 +46,9 @@ Provider Discovery Library: A library or piece of software that assists a DApp t
 Each wallet provider will be announced with the following interface `EIP6963ProviderInfo`. The values in the `EIP6963ProviderInfo` MUST be used as follows:
 
 - **`uuid`** - a globally unique of the wallet provider that MUST be ([UUIDv4][RFC 4122] compliant) to uniquely distinguish different [EIP-1193](./eip-1193.md) provider sessions. The cryptographic uniqueness provided by [UUIDv4][RFC 4122] guarantees that two independent `EIP6963ProviderInfo` objects can be separately identified.
-- **`name`** - A human-readable local alias of the wallet provider to be displayed to the user on the DApp. (e.g. `DopeWalletExtension` or `AwesomeWallet`)
-- **`icon`** - A [URI][RFC 3986] pointing to an image. Icon images MUST be square with 96x96px minimum resolution
-- **`rdns`** - an optional identifier that is used to optimistically identify the wallet by the Decentralized Application. If supplied by the wallet it MUST utilize an reverse DNS notation identifier that matches the domain name of the wallet provider.
+- **`name`** - a human-readable local alias of the wallet provider to be displayed to the user on the DApp. (e.g. `DopeWalletExtension` or `AwesomeWallet`)
+- **`icon`** - a [URI][RFC 3986] pointing to an image. Icon images MUST be square with 96x96px minimum resolution
+- **`rdns`** - an identifier that is used to optimistically identify the wallet by the Decentralized Application. If supplied by the wallet it MUST utilize an reverse DNS notation identifier that matches the domain name of the wallet provider.
```

Comment:
Forwarding from @kdenhartog’s message in the Telegram chat - might be good to add context of the failure case: 

“The Wallet MUST supply the rdns property. Similar to a user agent string in browsers, there are times where the supplied value could be unknown, invalid, incorrect, or attempt to imitate a different Wallet. Therefore, the DApp SHOULD be able to handle these failure cases with minimal degration to the functionality of the DApp.”

Reply:
👍🏻 yup, was going to add that in when I merged these changes in